packages/js-sdk/test/logging.test.mjs

```diff
@@ -0,0 +1,72 @@
+import { Session } from '../src'
+import { expect, test, vi } from 'vitest'
+import winston from 'winston';
```

Comment:
Are we adding winston just for one test?

Reply:
yea but only in `devDeps`, those does not count to the deps budget :D

Reply:
I don't see it there yet 😏

Reply:
i've added it to prev PR and forgot to cherry-pick it here, fixed 🙏